File: app/src/main/java/net/veldor/flibustaloader/database/entity/Bookmark.java
package net.veldor.flibustaloader.database.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import org.jetbrains.annotations.NotNull;

@Entity
public class Bookmark{
    @PrimaryKey(autoGenerate = true)
    public int id;

    @NotNull
    public String name = "";

    @NotNull
    public String link = "";
}

File: app/src/main/java/net/veldor/flibustaloader/database/entity/Bookmark.kt
package net.veldor.flibustaloader.database.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Bookmark {
    @PrimaryKey(autoGenerate = true)
    var id = 0
    @JvmField
    var name = ""
    @JvmField
    var link = ""
}